examples/custompopups/CustomAlertPopup.qml

```diff
@@ -20,42 +20,45 @@ AlertPopupInterface {
     Rectangle {
         anchors.fill: parent
         color: "lightsteelblue"
+        clip: flickable.contentHeight > height
 
-        Text {
-            id: title
-            anchors.bottom: message.top
-            anchors.bottomMargin: Theme.paddingLarge
-            width: parent.width
-            wrapMode: Text.WordWrap
-            horizontalAlignment: Text.AlignHCenter
-            font.bold: true
-            text: popup.title
-        }
-        Text {
-            id: message
-            anchors.centerIn: parent
-            width: parent.width
-            wrapMode: Text.WordWrap
-            horizontalAlignment: Text.AlignHCenter
-            text: popup.text
-        }
-        TextSwitch {
-            id: toggle
-            anchors.horizontalCenter: message.horizontalCenter
-            anchors.top: message.bottom
-            anchors.topMargin: Theme.paddingLarge
-            height: visible ? implicitHeight : 0
-            visible: popup.preventDialogsVisible
-            checked: popup.preventDialogsVisible && popup.preventDialogsPrefillValue
-            text: "Don't show this again"
-        }
-        Button {
-            id: acceptBtn
-            anchors.horizontalCenter: message.horizontalCenter
-            anchors.top: toggle.bottom
-            anchors.topMargin: Theme.paddingLarge
-            text: popup.acceptText
-            onClicked: { popup.accepted(); popup.visible = false }
+        SilicaFlickable {
+            id: flickable
+            anchors.fill: parent
+            contentHeight: content.height + Theme.paddingLarge
+
+            Column {
+                id: content
+                anchors.centerIn: parent
+                spacing: Theme.paddingLarge
```

Comment:
Without adding an explicit width, the Column will only grow to the largest element, which in this case is the button. It might be good to span the width of the parent by adding:
```
                width: parent.width - 2 * Theme.paddingLarge
```
Similarly for the other popups.

Reply:
Even though this is not fullscreen popup, changed to "width: parent.width - 2 * Theme.horizontalPageMargin"

Reply:
Looks much better, thanks.